doc/api/modules.md

```diff
@@ -291,6 +297,11 @@ Core modules are always preferentially loaded if their identifier is
 passed to `require()`. For instance, `require('http')` will always
 return the built in HTTP module, even if there is a file by that name.
 
+Core modules can also be identified using the `node:` prefix, in which case
+it bypasses the `require` cache. For instance, `require('node:http')` will
+always return the built in HTTP module, even if there is `require.cache` entry
+by that name.
```

Comment:
This is a bit of an odd example as the non-prefixed `require('http')` would be treated as a built in and bypass the require cache.

Reply:
currently builtin modules go through the cache if a matching entry exists, I don't understand the comment

Reply:
If there is a cache it should always point to the built in HTTP module, shouldn't it? AFAIK it's not possible for Node.js to return anything other than the built in module for built in modules.

Reply:
Actually, `require("http")` currently checks `require.cache["http"]` before calling `loadNativeModule("http")`.